website/static/_redirects

```diff
@@ -0,0 +1,5 @@
+# Redirects from what the browser requests to what we serve
+/docs/tasks/running-pods/       /docs/tasks/scheduling/
+/docs/provisioner-crd/	        /docs/provisioner/ 		
+/docs/faqs/	                    /docs/faq/
```

Comment:
nit: can we line this up?

Reply:
deferring this as it's not customer facing